models/public/mask_rcnn_inception_resnet_v2_atrous_coco/mask_rcnn_inception_resnet_v2_atrous_coco.md

```diff
@@ -0,0 +1,78 @@
+
+## Example
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Type                            | Instance segmentation                     |
+| GFlops                          | 675.314                                   |
+| MParams                         | 92.368                                    |
+| Source framework                | Tensorflow\*                              |
+
+## Performance
+
+## Input
+
+### Original Model
+
+Image, name: `image_tensor`, shape: [1x800x800x3], format: [BxHxWxC],
+   where:
+
+    - B - batch size
+    - H - image height
+    - W - image width
+    - C - number of channels
+
+   Expected color order: RGB.
+
+### Converted Model
+
+1. Image, name: `image_tensor`, shape: [1x3x800x800], format: [BxCxHxW],
+   where:
+
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
+
+   Expected color order: BGR.
+
+2. Information of input image size, name: `image_info`, shape: [1x3], format: [BxC],
+   where:
+
+    - B - batch size
+    - C - vector of 3 values in format [H,W,S], where H is an image height, W is an image width, S is an image scale factor (usually 1)
+
+## Output
+
+### Original Model
+
+1. Classifier, name: `detection_classes`. Contains predicted bounding boxes classes in a range [1, 91]. The model was trained on the Microsoft\* COCO dataset version with 90 categories of objects.
+2. Probability, name: `detection_scores`. Contains probability of detected bounding boxes.
+3. Detection box, name: `detection_boxes`. Contains detection boxes coordinates in a format `[y_min, x_min, y_max, x_max]`, where (`x_min`, `y_min`)  are coordinates of the top left corner, (`x_max`, `y_max`) are coordinates of the right bottom corner. Coordinates are rescaled to input image size.
+4. Detections number, name: `num_detections`. Contains the number of predicted detection boxes.
+5. Segmentation mask, name: `detection_masks`. Contains segmentation masks of detected objects for each class.
+
+### Converted Model
+
+1. The array of summary detection information, name: `reshape_do_2d`, shape: [1, 1, N, 7], where N is the number of detected
+bounding boxes. For each detection, the description has the format:
+[`image_id`, `label`, `conf`, `x_min`, `y_min`, `x_max`, `y_max`],
+    where:
+
+    - `image_id` - ID of the image in the batch
+    - `label` - predicted class ID
+    - `conf` - confidence for the predicted class
+    - (`x_min`, `y_min`) - coordinates of the top left bounding box corner (coordinates stored in normalized format, in range [0, 1])
+    - (`x_max`, `y_max`) - coordinates of the bottom right bounding box corner  (coordinates stored in normalized format, in range [0, 1])
+2. Segmentation mask, name: `masks`, shape: [N, 90, 33, 33], where N is the number of detected masks. 
```

Comment:
I think it is important to explain why 90 (not 91) that background class excluded from mask calculation.
And more correct description is segmentation heatmaps for all classes for every output bounding box